packages/remix-express/package.json

```diff
@@ -14,7 +14,8 @@
   "main": "dist/index.js",
   "typings": "dist/index.d.ts",
   "dependencies": {
-    "@remix-run/node": "1.12.0"
+    "@remix-run/node": "1.12.0",
+    "@remix-run/router": "1.3.1"
```

Comment:
Adapters will now include the router as a dependency so they can re-export  `MiddlewareContext`/`createMiddlewareContext` alongside their own `serverMiddleware` concepts